static/includes/AppInstallWizardACLConfiguration.md

```diff
@@ -0,0 +1,14 @@
+&NewLine;
+
+To set up permissions, select **Enable ACL** to show the ACL and ACE Entries options.
+
+First set the dataset permissions. Enter or browse to select the dataset path. 
+
+Next click **Add** to the right of **ACL Entries** to show the permissions settings.
+Set **ID Type** to **Entry is for a USER**, enter the UID for either the default apps user (**473**) or the UID for a new TrueNAS MinIO app admin user if one is created.
```

Comment:
I'm not understanding this.
If you mean the `apps` user on the host, the id is `568`. (and is also the default id set in the run-as option in the MinIO Enterprise)

But if you mean the minio user inside the container we should update the wording.
Stable has a default and hardcoded `473` user.
And enterprise has a default `568` but available for change.

Reply:
I'll update this content with this information and to clarify what UID is default for stable/community(?) apps and enterprise train apps. Also not sure if community apps will have a default user as they are submitting the yaml files so it could be anything or more likely to be a user they create on TrueNAS to be the administrator in their apps.